test/test-init.ts

```diff
@@ -58,124 +56,126 @@ function hasExpectedDependencies(packageJson: PackageJson): boolean {
+  // TODO: test generateConfigFile
+
+  // init
+  it('init should read local package.json', () => {
+    const originalContents = { some: 'property' };
+    return withFixtures(
+      { 'package.json': JSON.stringify(originalContents) },
+      async () => {
+        // TODO: this test causes `npm install` to run in the fixture directory.
+        // This may make it sensitive to the network, npm resiliency. Find a
+        // way to mock npm.
+        const result = await init.init(OPTIONS_YES);
+        assert.ok(result);
+        const contents = await readJson('./package.json');
+
+        assert.notStrictEqual(
+          contents,
+          originalContents,
+          'the file should have been modified'
+        );
+        assert.strictEqual(
+          contents.some,
+          originalContents.some,
+          'unrelated property should have preserved'
+        );
+      }
+    );
+  });
 
-// init
-test.serial('init should read local package.json', t => {
-  const originalContents = { some: 'property' };
-  return withFixtures(
-    { 'package.json': JSON.stringify(originalContents) },
-    async () => {
+  it('init should handle missing package.json', () => {
+    return withFixtures({}, async () => {
       // TODO: this test causes `npm install` to run in the fixture directory.
-      // This may make it sensitive to the network, npm resiliency. Find a
-      // way to mock npm.
+      // This may make it sensitive to the network, npm resiliency. Find a way to
+      // mock npm.
       const result = await init.init(OPTIONS_YES);
-      t.truthy(result);
+      assert.ok(result);
       const contents = await readJson('./package.json');
-
-      t.not(contents, originalContents, 'the file should have been modified');
-      t.is(
-        contents.some,
-        originalContents.some,
-        'unrelated property should have preserved'
-      );
-    }
-  );
-});
-
-test.serial('init should handle missing package.json', t => {
-  return withFixtures({}, async () => {
-    // TODO: this test causes `npm install` to run in the fixture directory.
-    // This may make it sensitive to the network, npm resiliency. Find a way to
-    // mock npm.
-    const result = await init.init(OPTIONS_YES);
-    t.truthy(result);
-    const contents = await readJson('./package.json');
-    t.truthy(hasExpectedScripts(contents));
-    t.truthy(hasExpectedDependencies(contents));
+      assert.ok(hasExpectedScripts(contents));
+      assert.ok(hasExpectedDependencies(contents));
+    });
   });
-});
 
-test.serial('init should support yarn', t => {
-  return withFixtures(
-    {
-      'package.json': JSON.stringify({ name: 'test' }),
-      'yarn.lock': '',
-    },
-    async () => {
-      const result = await init.init(OPTIONS_YARN);
-      t.truthy(result);
-
-      const contents = await readJson('./package.json');
-      t.truthy(contents.scripts.prepare === 'yarn run compile');
-    }
-  );
+  it('init should support yarn', async () => {
```

Comment:
Nit: this method has `async () => { return Promise }`, I've also seen `async () => { await Promise }` and `() => { return Promise }`, they're all valid but should we stick with just one pattern?